From 574b46a33be03fe7ab5aa1c342dee4ed6a278cb6 Mon Sep 17 00:00:00 2001
From: Mike Gerdts <mike.gerdts@joyent.com>
Date: Thu, 15 Feb 2018 01:40:02 +0000
Subject: [PATCH] OS-6628 bhyve needs to support at least 32 vcpus Reviewed by:
 John Levon <john.levon@joyent.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Reviewed by: Jerry Jelinek
 <jerry.jelinek@joyent.com> Approved by: John Levon <john.levon@joyent.com>

---
 usr/src/uts/i86pc/sys/vmm.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/usr/src/uts/i86pc/sys/vmm.h b/usr/src/uts/i86pc/sys/vmm.h
index cc87b8df30..757dc7ce7f 100644
--- a/usr/src/uts/i86pc/sys/vmm.h
+++ b/usr/src/uts/i86pc/sys/vmm.h
@@ -36,7 +36,7 @@
  * http://www.illumos.org/license/CDDL.
  *
  * Copyright 2015 Pluribus Networks Inc.
- * Copyright 2017 Joyent, Inc.
+ * Copyright 2018 Joyent, Inc.
  */
 
 #ifndef _VMM_H_
@@ -418,7 +418,7 @@ void vm_copyout(struct vm *vm, int vcpuid, const void *kaddr,
 int vcpu_trace_exceptions(struct vm *vm, int vcpuid);
 #endif	/* KERNEL */
 
-#define	VM_MAXCPU	16			/* maximum virtual cpus */
+#define	VM_MAXCPU	32			/* maximum virtual cpus */
 
 /*
  * Identifiers for optional vmm capabilities
-- 
2.21.0

